saleor/settings.py

```diff
@@ -237,6 +238,14 @@ def get_list(text):
 
 OPENEXCHANGERATES_API_KEY = os.environ.get('OPENEXCHANGERATES_API_KEY')
 
+# VAT configuration
+# Enabling vat requires valid vatlayer access key. By default prices entered in
+# dashboard are treated as net amounts. If you wish to reverse that and use
+# them as gross amounts instead change the BASE_PRICES_ARE_NET to False
+ENABLE_VAT = False
```

Comment:
How is this different from having a valid `VATLAYER_ACCESS_KEY`?

Reply:
I had a plan where this flag would disable whole tax handling in storefront to buy store admins time to setup things correctly in dashboard, or opt-out from the feature if it didn't work out for their store.

Reply:
But the mechanism to change it is the same as to remove the API keys?

Reply:
Thinking about this now, opt-in for VATLAYER could indeed be combination of having vatrates downloaded from API and `DEFAULT_COUNTRY` that has known standard vat rate.